src/controller.py

```diff
@@ -41,59 +42,155 @@
 
 def handle_drpc_request(drpc_request, connection_id):
     handler = {
-        "request_nonce": handle_drpc_request_nonce,
-        "request_attestation": handle_drpc_request_attestation,
+        "request_nonce": handle_drpc_request_nonce_v1,
+        "request_nonce_v2": handle_drpc_request_nonce_v2,
+        "request_attestation_v2": handle_drpc_request_attestation_v2,
     }.get(drpc_request["method"], handle_drpc_default)
 
     return handler(drpc_request, connection_id)
 
 
+def handle_drpc_response(drpc_response, connection_id):
+    handler = {
+        "request_attestation_v1": handle_drpc_request_attestation_v1,
```

Comment:
Should this key have the `_v1` suffix if it's from the old version? Or should it just stay as `"request_attestation"`

Reply:
Ya, when I tested manually it was added in. I think it comes from the v1 request nonce that triggers it.